### DISCUSSION TITLE
Upgrading versions

### DISCUSSION THREAD

**Discussion by onoufriosm** [TeselaGen/openVectorEditor#889] [Q&A]
Hi,

We are on an old version (10.1.61) and we want to upgrade to the latest version. 

We are just wondering if the sequence data schema is retrocompatible or if we need to do any migrations? (I haven't seen any breaking changes in the Changelog file to suggest this but wanted to ask to be on the safe side).

**Comment by onoufriosm** [comment]
@tnrich Any ideas on this? Thanks in advance.

**Comment by tnrich** [comment]
@onoufriosm you should be g2g but please do report if you hit any issues.

**Comment by tnrich** [comment]
Also, random side question but is your name `onoufriosm == Oh No You Frio (cold) Shake My (Head?)` lol I assume not but just checking...

**Comment by tnrich** [comment]
@onoufriosm any news? :)
